### PR TITLE
Feat/transfer nekoton logic

### PR DESCRIPTION
**Pull request by Alex-A4**
- Implement nekoton/external classes
- Update logic of converting `DynamicValue` in rust side
- Add converting `ErrorCode` to `anyhow::Error`
- Add lib-structure in dart side

# Question
Should we handle or log errors somehow where `throw ErroCode` happens?

### CODE REVIEW

File: packages/nekoton_bridge/native/src/utils/caller.rs [hunk 1]
```diff
@@ -16,16 +17,21 @@ use uuid::Uuid;
...
+    #[error("Network error")]
     Network,
+    #[error("Some error in logic")]
     Generic,
 }
 
 // / Dynamic value for transmitting between Dart and Rust. We can't use Box<dyn Any> because frb doesn't support it.
 #[derive(Clone, Debug)]
 pub enum DynamicValue {
+    U16(u16),
```
Comment (nesquikm) [inline packages/nekoton_bridge/native/src/utils/caller.rs:33]: maybe we should add
```dart
if (this is DynamicValue_U16) return (this as DynamicValue_U16).field0;
```
to dynamic_value.dart?

**Review by nesquikm — CHANGES_REQUESTED** [review]

File: packages/flutter_nekoton_bridge/lib/flutter_nekoton_bridge.dart [hunk 2]
```diff
@@ -89,11 +92,15 @@ Future<void> registerRustToDartCaller(RustToDartCaller rustToDartCaller) async {
...
-      ///   and how can we handle in rust side
+    } on Object catch (e) {
       if (id != null) {
-        lib.callSendResult(id: id, value: const DynamicValue.none());
+        /// To get here, just `throw ErrorCode.Network` for example
+        if (e is ErrorCode) {
+          lib.callSendResult(id: id, value: DynamicValue.error(e));
+        } else {
+          lib.callSendResult(
+              id: id, value: const DynamicValue.error(ErrorCode.Generic));
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/lib/flutter_nekoton_bridge.dart:102]: Agree, what do you think?

File: packages/nekoton_bridge/native/src/utils/caller.rs (hunk 1 above)
Comment (Alex-A4) [reply packages/nekoton_bridge/native/src/utils/caller.rs:33]: -

File: packages/flutter_nekoton_bridge/lib/flutter_nekoton_bridge.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (nesquikm) [reply packages/flutter_nekoton_bridge/lib/flutter_nekoton_bridge.dart:102]: All naming-related things are too complicated for me :) So, my suggestion is 'DartCallError'.

[the remark this answers is not in the document]
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/lib/flutter_nekoton_bridge.dart]: -

**Review by nesquikm — APPROVED** [review]

**Pull request merged**